auth0_flutter/example/web/index.html

```diff
@@ -38,6 +38,7 @@
   </script>
   <!-- This script adds the flutter initialization JS code -->
   <script src="flutter.js" defer></script>
+  <script src="sdk-2.0.js" defer></script>
```

Comment:
Can we use the CDN here?

Reply:
We can indeed. My thinking here was just to include it in the package to make it more robust, but considering CDN availability, caching, etc, we're probably better off including it from the CDN 👍🏻

Reply:
Fixed in https://github.com/auth0/auth0-flutter/pull/203/commits/573e301b4ccacf1cc4d078a19889c51ff1cd1c3a